Flaky test: org.keycloak.testsuite.forms.LoginTest#loginRememberMeExpiredMaxLifespan

## org.keycloak.testsuite.forms.LoginTest#loginRememberMeExpiredMaxLifespan
[Keycloak CI - Forms IT (chrome)](https://github.com/keycloak/keycloak/actions/runs/10790105322/job/29925593236) / [Pull Request #32631](https://github.com/keycloak/keycloak/pull/32631)

```
java.lang.AssertionError: Expected LoginPage but was AUTH_RESPONSE (https://localhost:8543/auth/realms/master/app/auth?state=4f7dd394-1aab-4a2c-abed-5254ea6ec872&session_state=9b878e27-2636-4834-9bfe-6fed70b51de2&iss=https%3A%2F%2Flocalhost%3A8543%2Fauth%2Frealms%2Ftest&code=62eec2a1-3538-4dc7-856b-f15053169fd8.9b878e27-2636-4834-9bfe-6fed70b51de2.ab56b74b-8333-4c48-a0be-6e6b48ed5a8b)
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.keycloak.testsuite.pages.AbstractPage.assertCurrent(AbstractPage.java:47)
	at org.keycloak.testsuite.pages.LoginPage.open(LoginPage.java:262)
...
```

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/LoginPage.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/LoginPage.java
@@ -22,6 +22,7 @@ import org.junit.Assert;
 import org.keycloak.common.util.Retry;
 import org.keycloak.testsuite.util.DroneUtils;
 import org.keycloak.testsuite.util.OAuthClient;
+import org.keycloak.testsuite.util.UIUtils;
 import org.keycloak.testsuite.util.WaitUtils;
 import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
@@ -247,10 +248,7 @@ public class LoginPage extends LanguageComboboxAwarePage {
     }
 
     public void setRememberMe(boolean enable) {
-        boolean current = rememberMe.isSelected();
-        if (current != enable) {
-            rememberMe.click();
-        }
+        UIUtils.switchCheckbox(rememberMe, enable);
     }
 
     public boolean isRememberMeChecked() {

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/UIUtils.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/UIUtils.java
@@ -1,6 +1,8 @@
 package org.keycloak.testsuite.util;
 
 import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.keycloak.common.util.Retry;
 import org.keycloak.testsuite.page.AbstractPatternFlyAlert;
 import org.openqa.selenium.By;
 import org.openqa.selenium.JavascriptExecutor;
@@ -98,6 +100,28 @@ public final class UIUtils {
     public static void click(WebElement element) {
         waitUntilElement(element).is().clickable();
         performOperationWithPageReload(element::click);
+    }
+
+    /**
+     * The method switches the checkbox to the expected state.
+     *
+     * It looks that since chrome 128, the single click sometimes does
+     * not work (See also similar issue {@link #clickLink(WebElement)}, so it is possible repeated multiple times until it reach
+     * the desired state
+     *
+     * @param checkbox Checkbox element to enable or disable
+     * @param enable If true, the checkbox should be switched to enabled (checked). If false, the checkbox should be switched to disabled (unchecked)
+     */
+    public static void switchCheckbox(WebElement checkbox, boolean enable) {
+        int maxAttempts = 4;
+
+        Retry.execute(() -> {
+            boolean current = checkbox.isSelected();
+            if (current != enable) {
+                UIUtils.click(checkbox);
+                Assert.assertNotEquals("Checkbox " + checkbox + " is still in the state " + current + " after click.", current, checkbox.isSelected());
+            }
+        }, maxAttempts, 0);
     }
 
     /**

